src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumChest.java

```diff
@@ -405,7 +457,11 @@ public <T> T getCapability(Capability<T> capability, EnumFacing side) {
             }
             return null;
         } else if (capability == CapabilityItemHandler.ITEM_HANDLER_CAPABILITY) {
-            return CapabilityItemHandler.ITEM_HANDLER_CAPABILITY.cast(combinedInventory);
```

Comment:
This change previously was to show the items in the output slot + the virtualized items to mods like AE2. Does this still happen with your changes?

If so, this `combinedInventory` is now unused.

Reply:
I forgot to check how AE2/TOP read the contents, so `combinedInventory` is still needed. I've added them back in now